File: hello-world.c
#include <stdio.h>
#include <string.h>

int main() {
   char msg1[10] = "hello, ";
   char msg2[10] = "world!";
   
   // Concatenate the strings
   strcat(msg1, msg2);
   
   // printf() displays the string inside quotation
   printf("%s\n",msg1);
   return 0;
}

File: hello-world.java
// Hello World Program

class HelloWorld {
    public static void main(String[] args) {
        String msg1 = "hello, ";
        String msg2 = "world!";
        
        // Prints out using println
        System.out.println(msg1.concat(msg2));
        
        // Joins the string using format
        String msg = String.format("%s%s",msg1,msg2);
        
        // Prints out using println
        System.out.println(msg.toString());
    }
}
